dialog/lib/dialog.ts

```diff
@@ -23,6 +23,8 @@ export const CLOSE_ACTION = 'close';
 const OPENING_TRANSITION_PROP = '--_opening-transition-duration';
 const CLOSING_TRANSITION_PROP = '--_closing-transition-duration';
 
+export type DialogTransition = 'grow'|'shrink'|'grow-down'|'grow-up'|'grow-left'|'grow-right';
```

Comment:
oh oof, we need jsdoc for this, just realized. Mind adding some? It can just say "Dialog transition type."